sdk/v2/authn/users.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/brigadecore/brigade/sdk/v2/restmachinery"
 )
 
+// UserLabel represents the canonical User label string
+const UserLabel = "User"
```

Comment:
Based on how this is being used, would `UserKind` be more appropriate?

Reply:
Same comment applies in several other places.

Reply:
Sure, shall I update all of these to `*Kind`?

Reply:
Yes... unless you disagree that my suggestion adds clarity to how those constants are used.